### PR TITLE
add full method 

### PR DESCRIPTION
**Pull request by Sillhouette**
* Add positive test for full method
* Add full method
* Add negative test for full method

### CODE REVIEW

**Review by dcmoore — APPROVED** [review]

File: src/board.py [hunk 1]
```diff
@@ -18,3 +18,5 @@ def stringify_spaces(self):
             #Board.border 
         ]
     
+    def full(self):
+        return not " " in self.spaces
```
Comment (tpetersen0308) [inline src/board.py:22]: What if you want to change the way an empty space is represented?

Comment (Sillhouette) [reply src/board.py:22]: Good point, I'll store it in a variable for the class to make it easier to change on a whim and make another pr. If the client wants the spaces represented in a completely different way then that's a discussion I need to have with them next time we chat

Comment (Sillhouette) [reply src/board.py]: This concern has been addressed in this PR:
https://github.com/Sillhouette/tic_tac/pull/8

Mind reviewing it real quick before I merge?

Comment (Sillhouette) [reply src/board.py:22]: I've made some adjustments to the infrastructure of this branch here: 

https://github.com/Sillhouette/tic_tac/pull/9

Please review for merge

**Pull request merged**